pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/TestKitUtil.java

```diff
@@ -82,71 +79,72 @@ public class TestKitUtil {
+    private String groupName;
+    private String kitTypeName;
+    private String studyGuid;
+    private String collaboratorPrefix;
+    private String userWithKitShippingAccess;
     @Getter
     @Mock
-    static EasyPostUtil mockEasyPostUtil = mock(EasyPostUtil.class);
+    EasyPostUtil mockEasyPostUtil = mock(EasyPostUtil.class);
     @Mock
-    static Address mockEasyPostAddress = mock(Address.class);
+    Address mockEasyPostAddress = mock(Address.class);
     @Mock static  Shipment mockEasyPostShipment = mock(Shipment.class);
     @Mock
-    static Parcel mockEasyPostParcel = mock(Parcel.class);
+    Parcel mockEasyPostParcel = mock(Parcel.class);
     @Mock
-    static PostageLabel mockParticipantLabel = mock(PostageLabel.class);
+    PostageLabel mockParticipantLabel = mock(PostageLabel.class);
     @Mock
-    static Tracker mockShipmentTracker = mock(Tracker.class);
+    Tracker mockShipmentTracker = mock(Tracker.class);
 
     public TestKitUtil(String instanceName, String studyGuid, String collaboratorPrefix, String groupName,
-                                  String kitTypeName) {
+                                  String kitTypeName, String kitTypeDisplayName) {
         this.instanceName = instanceName;
         this.studyGuid = studyGuid;
         this.collaboratorPrefix = collaboratorPrefix;
         this.groupName = groupName;
         this.kitTypeName = kitTypeName;
+        this.kitTypeDisplayName = kitTypeDisplayName;
     }
 
-    public static void deleteInstanceAndSettings() {
+    public void deleteGeneratedData() {
         SimpleResult results = inTransaction((conn) -> {
             SimpleResult dbVals = new SimpleResult();
             try {
-                delete(conn, "kit_dimension", "kit_dimension_id", kitDimensionId);
                 delete(conn, "kit_return_information", "kit_return_id", kitReturnId);
-                delete(conn, "carrier_service", "carrier_service_id", carrierId);
                 delete(conn, "ddp_kit_request_settings", "ddp_kit_request_settings_id", ddpKitRequestSettingsId);
+                delete(conn, "carrier_service", "carrier_service_id", carrierId);
+                delete(conn, "kit_dimension", "kit_dimension_id", kitDimensionId);
                 delete(conn, "ddp_instance_role", "ddp_instance_role_id", ddpInstanceRoleId);
-                delete(conn, "ddp_instance_group", "instance_group_id", ddpInstanceGroupId);
-                delete(conn, "ddp_instance", "ddp_instance_id", ddpInstanceId);
-                delete(conn, "ddp_group", "group_id", ddpGroupId);
-                cmiAdminUtil.deleteGeneratedData();
+                adminUtil.deleteGeneratedData();
             } catch (Exception e) {
                 dbVals.resultException = e;
             }
             return dbVals;
         });
+        if (results.resultException != null) {
+            throw new DsmInternalError("Error deleting instance and settings", results.resultException);
```

Comment:
These errors were being ignored, leading to downstream failures.  Once the exceptions became visible, it was clear that the rows needed to be deleted in a different order.